tests/test_shape.py

```diff
@@ -0,0 +1,157 @@
+import os
+import numpy as np
+from sklearn.utils.testing import (assert_array_equal,
+                                   assert_array_almost_equal,
+                                   assert_almost_equal)
+import oddt
+from oddt.shape import usr, usr_cat, electroshape, usr_similarity
+
+test_data_dir = os.path.dirname(os.path.abspath(__file__))
+benzene = oddt.toolkit.readstring('smi', 'c1ccccc1')
+benzene.make3D()
+benzene.removeh()
+methylo = oddt.toolkit.readstring('smi', 'c1ccccc1C')
+methylo.make3D()
+methylo.removeh()
+
+
+def test_usr():
+    if oddt.toolkit.backend == 'ob':
+        benzene_usr = np.array((1.394, 4.178e-07, 1.909e-10,
+                                1.734, 0.879, -0.582,
+                                1.735, 0.879, -0.583,
+                                1.735, 0.879, -0.583))
+        methylo_usr = np.array((1.578, 0.209, 0.0688,
+                                1.707, 0.767, -0.428,
+                                2.632, 1.961, -1.702,
+                                2.102, 1.561, 0.139))
+    else:
+        benzene_usr = np.array((1.392, 2.368e-15, 2.823e-22,
+                                1.732, 0.877, -0.580,
+                                1.732, 0.877, -0.580,
+                                1.732, 0.877, -0.580))
+        methylo_usr = np.array((1.572, 0.205, 0.064,
+                                1.697, 0.759, -0.420,
+                                2.614, 1.940, -1.660,
+                                2.095, 1.542, 0.117))
+
+    assert_array_almost_equal(usr(benzene), benzene_usr, decimal=3)
+    assert_array_almost_equal(usr(methylo), methylo_usr, decimal=3)
+
+
+def test_usr_cat():
+    if oddt.toolkit.backend == 'ob':
```

Comment:
USR CAT should be the same, Electroshape will differ due to partial charges being different.